tree/dataframe/test/datasource_ntuple.cxx

```diff
@@ -47,14 +47,26 @@ TEST_F(RNTupleDSTest, ColTypeNames)
    RNTupleDS tds(std::move(fPageSource));
 
    auto colNames = tds.GetColumnNames();
-   ASSERT_EQ(colNames.size(), 5);
+   ASSERT_EQ(7, colNames.size());
 
    EXPECT_TRUE(tds.HasColumn("pt"));
    EXPECT_TRUE(tds.HasColumn("energy"));
+   EXPECT_TRUE(tds.HasColumn("#nnlo"));
    EXPECT_FALSE(tds.HasColumn("Address"));
 
    EXPECT_STREQ("std::string", tds.GetTypeName("tag").c_str());
    EXPECT_STREQ("float", tds.GetTypeName("energy").c_str());
+   EXPECT_STREQ("ROOT::Experimental::ClusterSize_t::ValueType", tds.GetTypeName("#jets").c_str());
+}
+
+
+TEST_F(RNTupleDSTest, CardinalityColumn)
+{
+   auto df = ROOT::Experimental::MakeNTupleDataFrame(fNtplName, fFileName);
+
+   // Check that the special column #<collection> works with jitting
+   auto max_njets = df.Define("njets", "@jets.size").Max("njets");
```

Comment:
```suggestion
   auto max_njets = df.Define("njets", "#jets").Max("njets");
```

Reply:
With respect to the tokenizer, that doesn't help us unfortunately. The `@` symbol is equally invalid syntax.

Reply:
yes yes, just to remind ourselves to change it back to `#`